fuzzers/qemu_launcher/src/fuzzer.rs

```diff
@@ -91,8 +90,8 @@ pub struct FuzzerOptions {
     #[arg(long, help = "Output directory")]
     output: String,
 
-    #[arg(long, help = "Timeout in seconds", default_value_t = 1_u64)]
-    timeout: u64,
+    #[arg(long, help = "Timeout in seconds", default_value = "1", value_parser = FuzzerOptions::parse_timeout)]
```

Comment:
Any reason not to use millis here as well, as we do everywhere else?

Reply:
I don’t feel strongly one way or another. Didn’t realise there was a convention. I’ll change it up.